packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_maps_flutter_android.dart

```diff
@@ -1146,6 +1146,20 @@ PlatformZoomRange? _platformZoomRangeFromMinMaxZoomPreferenceJson(
   return PlatformZoomRange(min: minMaxZoom[0], max: minMaxZoom[1]);
 }
 
+/// Converts platform interface's JointType to Pigeon's PlatformJointType.
+@visibleForTesting
+PlatformJointType platformJointTypeFromJointType(
+    JointType jointType) {
+  switch (jointType) {
+    case JointType.mitered:
+      return PlatformJointType.mitered;
+    case JointType.bevel:
+      return PlatformJointType.bevel;
+    case JointType.round:
+      return PlatformJointType.round;
+  }
```

Comment:
This needs a fallback to prevent undefined behavior if the platform interface gets a new value; see https://github.com/flutter/packages/blob/0e8c5da679c8c812ee6af457a7768a6e828e6308/packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_maps_flutter_android.dart#L992-L998

Reply:
This has not been addressed; if a value is added to `JointType` in the platform interface package, someone running the above version of this package's code with the new platform interface version would have undefined behavior here.